### PR TITLE
Create ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-56446](https://cumulocity.atlassian.net/browse/MTM-56446)
Original PR: [6737](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6737)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: the branding editor was sometimes running into issues (#4918) [GRAFT][release/y2024] (#6737)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed an issue with the branding editor sometimes not working correctly
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56446
+version: 1018.503.107
+---
+the branding editor was sometimes running into issues (#4918) [GRAFT][release/y2024] (#6737)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some cases, the branding editor in the Cumulocity IoT platform was not working as expected and users experienced problems when trying to edit the branding. This issue has now been resolved. With this fix, users can reliably use the branding editor again to customize the look and feel of their Cumulocity IoT experience without running into any problems.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56446
+version: 1018.503.107
+---
+In some cases, the branding editor in the Cumulocity IoT platform was not working as expected and users experienced problems when trying to edit the branding. This issue has now been resolved. With this fix, users can reliably use the branding editor again to customize the look and feel of their Cumulocity IoT experience without running into any problems.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md:17]:
```suggestion
In some cases, the Branding editor in the {{< product-c8y-iot >}} platform was not working as expected and users experienced problems when trying to edit the branding. This issue has now been resolved and users can reliably use the Branding editor again to customize the look and feel of their {{< product-c8y-iot >}} experience.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed an issue with the branding editor sometimes not working correctly
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-107-the-branding-editor-was-sometimes-running-into-issues-4918-graft-release.md:3]:
```suggestion
title: Branding editor works reliably again
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**